CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- LSP: `RustAnalyzer reload_settings` command to reload settings without restarting.
```

Comment:
suggestion: For consistency with the `RustLsp` commands, I would suggest camel case.